README.md

```diff
@@ -15,6 +15,8 @@ The Timestream JDBC driver is distributed as two JAR files.
 ### JAR With All Dependencies Shaded Except SLF4J (amazon-timestream-jdbc-\<version\>-jar-with-all-dependencies.jar)
 This is the shaded JAR which bundles up all the required dependencies and can be used directly in Java projects and external applications, without explicitly adding external dependencies. This JAR does not relocate the classpath of SLF4J, allowing logging to be enabled in external applications.
 
+- Instructions for using fully shaded JAR file with [SQuirreL](markdown/setup/squirrel-driver-setup.md)
```

Comment:
Adjust based on example here - https://github.com/Bit-Quill/amazon-timestream-driver-jdbc/pull/2/files#r1049204514

Reply:
I can work on this with documentation PR